lib/tasks/b2b.rake

```diff
@@ -0,0 +1,19 @@
+namespace :b2b do
+  desc 'Create corporate article type'
+  task create_type: :environment do
+    Rails.logger = ActiveRecord::Base.logger = Logger.new(STDOUT)
+
+    site = Comfy::Cms::Site.first
+    layout = site.layouts.find_or_initialize_by(
+      label: 'Corporate Article',
+      identifier: 'corporate_article',
+      content: "{{ cms:page:content:rich_text }}\n"
+    )
+
+    Rails.logger.info("*" * 80)
+    Rails.logger.info("Saving ##{layout.identifier} for site ##{site.label} and mirroring to others sites.")
+    Rails.logger.info("*" * 80)
+
+    Rails.logger.info("Layout saved successfuly. Details: \n#{layout.inspect}.\nMirrors: #{layout.mirrors}") if layout.save
```

Comment:
It irks me slightly when I see a long line and then a surprise statement modifier at the end. I've heard them called 'if surprises' before. Which seems pretty apt to me. 

One way to solve this is by instead saying `layout.save && Rails.logger.info(...)` but you might find that offensive too depending on your leanings.